src/extension.ts

```diff
@@ -122,12 +121,17 @@ function toUrl(filePath) {
   return encodeURI("file://" + pathName);
 }
 
-function resolve(tsdkPathSetting) {
+function resolveInAnyWorkspaceFolder(tsdkPathSetting) {
   if (Path.isAbsolute(tsdkPathSetting)) {
-    return tsdkPathSetting;
+    return FS.existsSync(tsdkPathSetting) ? tsdkPathSetting : undefined;
   }
-
-  return Path.join(VSCode.workspace.rootPath, tsdkPathSetting);
+  for (let folder of (VSCode.workspace.workspaceFolders || [])) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Identifier 'folder' is never reassigned; use 'const' instead of 'let'. ([typescript:S3353](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS3353))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarlint.vscode%3Asonarlint-vscode&pullRequest=16&issues=AWMHuAuKqwcY0PgyxSYq&open=AWMHuAuKqwcY0PgyxSYq)

[ISSUE_KEY]: AWMHuAuKqwcY0PgyxSYq